spec/models/job_spec.rb

```diff
@@ -54,4 +54,43 @@
       expect(job.author).to eq author
     end
   end
+
+  describe ".search" do
+    let(:matching_description_job) do
+      create(
+        :job,
+        :published,
+        description: "Familiarity with agile methodologies, especially Scrum or Kanban"
+      )
+    end
+
+    let(:matching_title_job) do
+      create(
+        :job,
+        :published,
+        title: "Fullstack developer"
+      )
+    end
+
+    let(:not_matching_job) do
+      create(
+        :job,
+        :published,
+        title: "Fullstack developer",
+        description: "Familiarity with agile methodologies, especially Scrum or Kanban"
+      )
+    end
+
+    it "returns a job when the query matches its title" do
+      expect(subject.class.search("fullstack")).to include matching_title_job
```

Comment:
`subject` can be defined the same way as `let`. 

Example:
```
subject { described_class.search('fullstack') }

it 'does something'
  expect(subject).to eq 5
end
```

Having the subject executing what you test allows the short implicit form. 

```
it { is_expected.to include matching_title_job }
```

😹